en/components/general/cli/getting-started-with-angular-schematics.md

```diff
@@ -0,0 +1,157 @@
+
+To create an application that is configured to use the Ignite UI for Angular controls, run the `ng new` command providing `@igniteui/angular-schematics` to the collection option.
+
+```cmd
+ng new newAngularProject --collection="@igniteui/angular-schematics" --template=side-nav
+```
+
+The new application is created in a directory with the same name (`newAngularProject`). There are several project templates from which you can choose when creating an Ignite UI for Angular application:
+
+| template id   | template description |
+| ---           | ---                  |
+| empty         | Project structure with routing and a home page |
+| side-nav      | Project structure with side navigation drawer |
+| side-nav-auth | Side navigation project extended with user authentication module. <br> [Angular Authentication Project Template](auth-template.md) topic covers the project template in detail. |
+
+Additionally, you can specify **arguments** to control the theme or skip packages install:
+
+<details>
+  <summary><u>name</u></summary>
+  <p>
+    <code>name</code> (alias: <code>-n</code>)
+  </p>
+  <p>
+    The name of the application. The application is created inside a directory with the same name.
+  </p>
+</details>
+
+<details>
+  <summary><u>framework</u><span align="right"><strong> (Ignite UI CLI only)</strong></span></summary>
+  <p>
+    <code>--framework</code> (alias: <code>-f</code>) <em>default value: "jquery"</em>
+  </p>
+  <p>
+    Framework to setup project for. The supported frameworks are jQuery, Angular and React.
+  </p>
+</details>
+
+<details>
+  <summary><u>type</u><span align="right"><strong> (Ignite UI CLI only)</strong></span></summary>
+  <p>
+    <code>--type</code> (alias: <code>-t</code>)
+  </p>
+  <p>
+    The available project types depend on the selected framework.
+  </p>
+</details>
+
+<details>
+  <summary><u>theme</u></summary>
+  <p>
+    <code>--theme</code> (alias: <code>-th</code>)
+  </p>
+  <p>
+    Project theme (depends on project type).
+  </p>
+</details>
+
+<details>
+  <summary><u>skip-git</u></summary>
+  <p>
+    <code>--skip-git</code> (alias: <code>--sg</code>)
+  </p>
+  <p>
+    When this option is used, the automatic repository initialization with Git will be skipped. If the option is omitted, then the global skip-git configuration property is used.
+  </p>
+</details>
+
+<details>
+  <summary><u>skip-install</u></summary>
+  <p>
+    <code>--skip-install</code> (alias: <code>--si</code>)
+  </p>
+  <p>
+    The <code>new</code> command will install package dependencies on project creation. Passing this flag will skip the initial installation.
+  </p>
+</details>
+
+<details>
+  <summary markdown='span'><u>template</u></summary>
+  <p>
+    <code>--template</code>
+  </p>
+  <p>
+    Use this option if there are different project templates for a specific framework type. 
+    Currently this option is available only for Ignite UI for Angular igx-ts project types.</p>
+</details>
+
```

Comment:
If you already have an Angular project, created without providing the `@igniteui/angular-schematics` collection, like it is above, you can add the `igntieui-angular` product, using the following command:

```cmd
ng add igniteui-angular
```